xls/netlist/parse_netlist_main_test.sh

```diff
@@ -14,11 +14,16 @@
 # limitations under the License.
 
 
-# Find input files
+# Test with an cell library
 echo 'entries: { kind: INVERTER, name: "INV" input_names: "A" output_pin_list { pins { name: "ZN" function: "F" } } }' > "${TEST_TMPDIR}/fake_cell_library.textproto"
 echo 'module main(i, o); input i; output o; INV inv_0(.A(i), .ZN(o)); endmodule' > "${TEST_TMPDIR}/netlist.v"
 
 BINPATH=./xls/netlist/parse_netlist_main
 $BINPATH "${TEST_TMPDIR}/netlist.v" "${TEST_TMPDIR}/fake_cell_library.textproto" || exit -1
 
+# Test without a cell library
+echo 'module main(a0, a1, a2, a3, q0); input a0, a1, a2, a3; output q0; SB_LUT4 #(.LUT_INIT(16'"'"'h8000)) q0_lut (.I0(a0), .I1(a1), .I2(a2), .I3(a3), .O(q0)); endmodule' > "${TEST_TMPDIR}/netlist2.v"
+
+$BINPATH "${TEST_TMPDIR}/netlist2.v" || exit -1
```

Comment:
if you make the first line:

#!/bin/bash -e

Then the script will fail if any of the commands inside fail. That way you don't need to add the "|| -1" which is easy to forget.

Reply:
done